mlflow/pyspark/ml/__init__.py

```diff
@@ -666,6 +666,7 @@ def autolog(
     exclusive=False,
     disable_for_unsupported_versions=False,
     silent=False,
+    log_eval_metrics=True,
```

Comment:
```suggestion
    log_post_training_metrics=True,
```

Should this param be renamed to `log_post_training_metrics`?

Reply:
Yep! Sounds good to me!